packages/components/Menu/src/MenuItemRadio/MenuItemRadio.styling.ts

```diff
@@ -0,0 +1,97 @@
+import { Platform } from 'react-native';
+
+import type { Theme, UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, fontStyles, layoutStyles } from '@fluentui-react-native/tokens';
+
+import type { MenuItemRadioProps, MenuItemRadioTokens, MenuItemRadioSlotProps } from './MenuItemRadio.types';
+import { menuItemRadioName } from './MenuItemRadio.types';
+import { defaultMenuItemRadioTokens } from './MenuItemRadioTokens';
+
+export const menuItemRadioStates: (keyof MenuItemRadioTokens)[] = ['hovered', 'focused', 'pressed', 'disabled', 'checked'];
+
+export const stylingSettings: UseStylingOptions<MenuItemRadioProps, MenuItemRadioSlotProps, MenuItemRadioTokens> = {
+  tokens: [defaultMenuItemRadioTokens, menuItemRadioName],
```

Comment:
I think this file could be forked between android and non-android since the non-android styles align to menu item?

Reply:
i guess only having  MenuItemRadio.styling.android.ts and not having MenuItemRadio.styling.ts makes import work differently like : import { stylingSettings } from './MenuItemRadio.styling.android';
 
 But this will be on MenuItemRadio.android.tsx that is android specific file so i am ok to have it as creating MenuItemRadio.styling.ts which doesn't do anything doesn't make sense.